tests/test_segmentation_display.py

```diff
@@ -22,3 +23,72 @@ def test_segmentation_opacity_can_be_toggled_by_keypress(
 
     if render_interactive:
         a_slice_view.interactor().Start()
+
+
+def test_segmentation_opacity_modifications(
```

Comment:
This test should be broken into 3:
* Test opacity 2D
* Test opacity 3D
* Test opacity mode

Common code can be put in a fixture.

@pytest.mark.parametrize can be used for the opacity mode.

Also, the segmentation model has only one segment_id. So the code can be simplified